src/aegis_ai/features/__init__.py

```diff
@@ -21,39 +21,49 @@ class Feature:
     def __init__(self, agent: Agent):
         self.agent = agent
 
-    async def _timeout_wrap(self, prompt, **kwargs):
```

Comment:
cool - this is being removed solely because of `except asyncio.TimeoutError:` handling on line 42